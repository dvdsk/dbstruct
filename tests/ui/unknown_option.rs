use dbstruct::dbstruct;

#[dbstruct(starship=enterprise)]
struct Test {
    field: Option<u8>,
}

fn main() {}
