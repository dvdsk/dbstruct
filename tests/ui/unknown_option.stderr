error: not a known dbstruct option
 --> tests/ui/unknown_option.rs:3:12
  |
3 | #[dbstruct(starship=enterprise)]
  |            ^^^^^^^^
  |
  = help: the only supported option currently is: db
